content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -124,7 +124,7 @@ Each path in an Ingress has a corresponding path type. There are three supported
 path types:
 
 * _`ImplementationSpecific`_ (default): With this path type, matching is up to
-  the IngressClass. Implementations can treat this as a separate `pathType or
+  the IngressClass. Implementations can treat this as a separate `pathType` or
```

Comment:
Thanks @pmalmgren for this pull request.
I am not convinced that the emphasis is needed.
/lgtm